api/src/args.rs

```diff
@@ -2,40 +2,40 @@ use std::net::IpAddr;
 use std::path::PathBuf;
 use std::str::FromStr;
 
+use clap::Parser;
 use fqdn::FQDN;
 use shuttle_common::Port;
-use structopt::StructOpt;
 
-#[derive(StructOpt)]
-#[structopt(name = "shuttle")]
+#[derive(Parser)]
+#[clap(name = "shuttle")]
 pub struct Args {
-    #[structopt(long, about = "Override the default root path for shuttle")]
+    #[clap(long, help = "Override the default root path for shuttle")]
```

Comment:
Since we are doing this for consistency, I'm wondering if it is also worthwhile to replace the `help` attribute with a doc comment like provisioner has them?

Reply:
Sure thing!

Reply:
Done!